src/components/List/ListItem/ListItem.story.jsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import { Edit16, Star16, StarFilled16 } from '@carbon/icons-react';
+
+import { Button, OverflowMenu, OverflowMenuItem } from '../../..';
+
+import ListItem from './ListItem';
+
+storiesOf('Watson IoT Experimental|ListItem', module)
```

Comment:
i see we have the first story with knobs. Can we eliminate some of these stories if they can be represented with Knobs?

Reply:
Could you take a pass at this @davidicus?  Would be best I think to have the component library caretakers drive consistency in how we do our IoT stories, at least until we have a solid pattern to follow 👍